patches/chrome-browser-chrome_browser_main.cc.patch

```diff
@@ -10,12 +10,12 @@ index 331bc5accd56a16da324cc25360786b0b78a8aa7..cc1229861f77d0bf64dc00035e18e827
  #include "build/build_config.h"
  #include "cc/base/switches.h"
  #include "chrome/browser/about_flags.h"
-@@ -1009,7 +1010,7 @@ int ChromeBrowserMainParts::PreEarlyInitialization() {
-   scoped_refptr<base::SequencedTaskRunner> local_state_task_runner =
-       CreateLocalStateTaskRunner();
-   browser_process_ =
--      std::make_unique<BrowserProcessImpl>(local_state_task_runner.get());
-+      std::make_unique<BraveBrowserProcessImpl>(local_state_task_runner.get());
+@@ -989,7 +990,7 @@ int ChromeBrowserMainParts::PreEarlyInitialization() {
+   for (size_t i = 0; i < chrome_extra_parts_.size(); ++i)
+     chrome_extra_parts_[i]->PreEarlyInitialization();
+ 
+-  browser_process_ = std::make_unique<BrowserProcessImpl>();
++  browser_process_ = std::make_unique<BraveBrowserProcessImpl>();
```

Comment:
this is fine, but I thought you were going to replace these with chromium_src overrides?

Reply:
Not added here, will be doing it in a followup.

Reply:
sounds good

Reply:
PR'ed.